include/vast/Dialect/LowLevel/Passes.td

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022-present, Trail of Bits, Inc.
+
+#ifndef VAST_DIALECT_LOWLEVEL_PASSES_TD
+#define VAST_DIALECT_LOWLEVEL_PASSES_TD
+
+include "mlir/Pass/PassBase.td"
+
+def ToLLVM : Pass<"vast-ll-to-llvm", "mlir::ModuleOp"> {
```

Comment:
Rename to LLToLLVM

Reply:
Will do `CoreToLLVM` as this unfortunately requires the module to be in hybrid state of `ll + hl` right now.